lib/mastodon/feature.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Mastodon::Feature
+  class << self
+    def enabled_features
+      @enabled_features ||=
+        (ENV['EXPERIMENTAL_FEATURES'] || '').split(',').map(&:strip)
+    end
+
+    def method_missing(name)
+      if respond_to_missing?(name)
+        feature = name.to_s.sub(/_enabled\?$/, '')
```

Comment:
More readable I think and a tiny bit faster:
```suggestion
        feature = name.to_s.delete_suffix('_enabled?')
```

Reply:
Thanks, that is brilliant. I keep forgetting these methods exist. I mean they have only been introduced ... *checks notes* ... 7 years ago :man_facepalming: :older_man: